charts/artifact-hub/values.schema.json

```diff
@@ -417,27 +417,27 @@
                                     "type": "object",
                                     "properties": {
                                         "enabled": {
-                                            "title": "Enable Github oauth",
+                                            "title": "Enable GitHub OAuth",
```

Comment:
Brand (this presumably requires a version bump to the chart)

Reply:
Yes, can you please bump it to `1.6.1-1`?